internal/resources/cluster/cluster_full_name.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var ClusterFullname = &schema.Schema{
 	Type:        schema.TypeList,
-	Description: "The schema for cluster iam policy full name",
+	Description: "The schema for cluster full name",
```

Comment:
Since this change is done, it should be reflected in docs too as the schema description change. Also, please run the acceptance test for IAM policy and cluster resource for this.

Reply:
Update the docs

`

=== RUN   TestAcceptanceForIAMPolicyResource
    resource_iam_policy_test.go:187: start IAM policy resource acceptance tests!
TMC resources applied to the cluster successfully
    resource_iam_policy_test.go:234: IAM policy resource acceptance test complete for single resource!
    resource_iam_policy_test.go:296: IAM policy resource acceptance test complete for multiple resources!
    resource_iam_policy_test.go:297: all IAM policy resource acceptance tests complete!
--- PASS: TestAcceptanceForIAMPolicyResource (253.90s)

`

`

=== RUN   TestAcceptanceForAttachClusterResource
TMC resources applied to the cluster successfully
    resource_cluster_test.go:72: cluster resource acceptance test complete!
--- PASS: TestAcceptanceForAttachClusterResource (420.21s)
=== CONT  TestGetNodepoolForCluster

`